site/styles/_forms.scss

```diff
@@ -62,9 +63,11 @@ s72-activatedevice-form,
 .s72-error-message {
   background: $error-background;
   color: $error-color;
-  font-weight: $font-weight-normal;
+  letter-spacing: .2px;
```

Comment:
If you are trying to [adjust the tracking](https://silvawebdesigns.com/how-to-calculate-css-letter-spacing-vs-tracking-in-typography/), may I suggest `em` over `px`?